metacat-common-server/src/main/java/com/netflix/metacat/common/server/events/MetacatEventBus.java

```diff
@@ -79,22 +77,29 @@ public void postAsync(final ApplicationEvent event) {
     public void postSync(final ApplicationEvent event) {
         log.debug("Received request to post an event {} synchronously", event);
         this.eventSyncCounter.increment();
-        this.eventPublisher.publishEvent(event);
+        super.multicastEvent(event);
     }
 
-    /**
-     * Registers an object.
-     *
-     * @param object object
-     */
-    public void register(final Object object) {
+    @Override
+    public void addApplicationListener(final ApplicationListener listener) {
+        super.addApplicationListener(listener);
+        asyncEventMulticaster.addApplicationListener(listener);
     }
 
-    /**
-     * De-registers an object.
-     *
-     * @param object object
-     */
-    public void unregister(final Object object) {
+    @Override
+    public void removeApplicationListener(final ApplicationListener listener) {
+        super.removeApplicationListener(listener);
+        asyncEventMulticaster.removeApplicationListener(listener);
+    }
+
+    @Override
+    public void removeAllListeners() {
+        super.removeAllListeners();
+        asyncEventMulticaster.removeAllListeners();
+    }
+
+    @Override
+    public void multicastEvent(final ApplicationEvent event) {
```

Comment:
Is there a point to having this in here? If you don't put this method in here doesn't it just call the super anyway